worker.go

```diff
@@ -45,8 +45,17 @@ type Worker struct {
 
 func setSkipTLS(o *mqtt.ClientOptions) {
 	oldTLSCfg := o.TLSConfig
-	oldTLSCfg.InsecureSkipVerify = true
-	o.SetTLSConfig(oldTLSCfg)
+
+	if oldTLSCfg == nil {
+		o.SetTLSConfig(&tls.Config{
+			// InsecureSkipVerify = verify that cert contents
+			// match server. IP matches what is in cert etc.
```

Comment:
That comment does not reflect the reality. If set InsecureSkipVerify, the certificate is not checked for validity. Instead every TLs certificate is accepted.

Reply:
Ah sorry, that was a stupid c/p error